includes/core/classes/class-setup.php

```diff
@@ -432,6 +432,11 @@ public function check_users_can_register(): void {
 	 */
 	public function check_gatherpress_alpha(): void {
 		if (
+			// While a plugin from w.org would have a unigue slug to check against,
+			// it different for .zips downloaded from github.com.
+			// A download via "<> Code ⏷" > "Download ZIP" will include the branch in the file name.
+			is_plugin_active( 'gatherpress-alpha-main/gatherpress-alpha.php' ) ||
+			// A download via "Releases" will provide a zip without a branch name attached.
```

Comment:
Might be better to change this to see if the GatherPress Alpha version constant is defined, then we don't have to worry about file name.

Reply:
Good idea.